scripts/build.js

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+require('shelljs/global');
+
+var coveralls = require('coveralls');
+var helpers = require('./helpers');
+
+var Module = helpers.Module;
+var run = helpers.run;
+var git = helpers.git;
+
+var TRAVIS_BRANCH = process.env.TRAVIS_BRANCH;
+var IS_PULL_REQUEST = process.env.TRAVIS_PULL_REQUEST === 'true';
+var TEST_ONLY = process.env.TEST_ONLY === 'true';
+
+var modules = Module.getUpdated();
+
+if (!modules.length) {
+  echo('No code changes found, exiting early.');
+  exit();
```

Comment:
Per [shelljs](https://github.com/shelljs/shelljs) I don't think so. I'm guessing it calls `process.exit()` underneath.